DevREADME.md

```diff
@@ -99,4 +99,132 @@ You may encounter the failure of npm install, the error message would say:
 Error installing vscode.d.ts: Error: read ECONNRESET
 If you encounter this problem, maybe your NODE is too old, in version v10.13.0 it 
 can work well.
-or run "npm cache clean -force"
\ No newline at end of file
+or run "npm cache clean -force"
+
+---
```

Comment:
it only needs to paste the github link to this file? Or we can adjust the guide lines. I think anyway we should paste the github link for the copyright

Reply:
I don't recall seeing a copyright on the minimalist version. Plus I did alter it quite a bit already with the intent for us to alter it a great deal more.

Reply:
I went back through their wiki pages (all 2'ish of them) and there are absolutely no copyrights posted within their wiki. Their code base is licensed under "Revised BSD" and each of their actual *.TS files do in fact contain a copyright. 
- Maybe they also borrowed that wiki page conent as a starting point from somewhere and thus can't put their copyright on it? 
- Maybe this type of documentation (especially this abstract) isn't worth putting a copyright on? 

I am clearly not familiar with licenses in general, but this content seems very standard to the vast majority and highly subjective personal preference. I would be surprised if this winds up even 25% equal to the original by the time we are done injecting our own ideas and restructuring things.